index.html

```diff
@@ -4240,47 +4240,54 @@ <h2>HTTP</h2>
+specification always express JSON-LD, RDF processing is not required to be
+performed, since JSON-LD is a concrete RDF syntax as described in [RDF11-CONCEPTS]. 
+Hence, a JSON-LD document is both an RDF document and a JSON document and correspondingly represents an instance of an RDF data model.
+See <a href="JSON-LD11##relationship-to-rdf">Relationship to RDF</a> for more details.
+          </p>
+          <p>
+Some scenarios where processing a <a>verifiable credential</a> or a <a>verifiable presentation</a> as RDF is
 desirable include, but are not limited to:
           </p>
-
           <ul>
             <li>
-Before securing or after verifying content
-that requires <a href="https://csrc.nist.gov/glossary/term/data_integrity">data
-integrity</a>, such as a
-<a>verifiable credential</a> or <a>verifiable presentation</a>.
+Before securing or after verifying <a>verifiable credentials</a> or <a>verifiable presentations</a>,
+in cases where the issuer has leveraged the <code>@context</code> to provide details regarding 
+their intended expression of the <a href="#verifiable-credential-graphs">Verifiable Credential Graphs</a>.
             </li>
             <li>
-When performing JSON Schema validation, as described in Section
-<a href="#data-schemas"></a>.
+After performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>, in order to mitigate attacks 
+on the canonicalization process as described in <a href="https://w3c.github.io/rdf-canon/spec/#dataset-poisoning">Dataset Poisoning</a>
             </li>
             <li>
 When serializing or deserializing <a>verifiable credentials</a> or
-<a>verifiable presentations</a> into systems that store or index their contents.
+<a>verifiable presentations</a> into systems that store or index claims about subject in
+<code>text/turtle</code>, <code>application/n-quads</code>, <code>application/ld+json</code>. 
             </li>
             <li>
-When operating on <a>verifiable credentials</a> or <a>verifiable
-presentations</a> in a software application, after verification or validation
-is performed for securing mechanisms that require an understanding of
-and/or processing of JSON-LD.
+When validating <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application, after verification 
+is performed for securing mechanisms that do not require an understanding
+or processing of JSON-LD.
             </li>
             <li>
-When an application chooses to process the media type using the `+json`
+When an application chooses to process the media type using the `+ld+json`
 structured media type suffix.
             </li>
           </ul>
 
           <p>
-That is, JSON processing is allowed as long as the document being consumed or
-produced is a <a>conforming document</a>. If JSON processing is desired, an
-implementer is advised to follow the following rule:
+Naive JSON processing is allowed as long as the document being consumed or
```

Comment:
"Naive JSON processing" isn't defined, so it's not clear what is being talked about here. Suggest we point this to the renamed "JSON Processing" section and call it "Minimal Processing" (under the JSON-LD Processing section).

Reply:
@OR13, please respond to this.

Reply:
I don't support any form of encouraging implementers of the spec to ignore / not understand conforming documents.

I think the language here is effective, but perhaps not clear enough, and could be fixed in an editorial pass in the future.

Reply:
I have tried to use "application-specific", which you use in your prose. No need to delay clarification. There is a concrete suggestion there, are you taking it, offering an alternative, or rejecting it?

Reply:
I have another suggestion here that @OR13 said he would accept:

https://github.com/w3c/vc-data-model/pull/1302/files#r1347713087

Can you apply that suggestion, @OR13?